Eatery Blue/UI/SettingsMain/SettingsMainMenuLoginStatusView.swift

```diff
@@ -26,20 +25,23 @@ class SettingsMainMenuLoginStatusView: UIView {
     private func setUpSelf() {
         addSubview(label)
         setUpLabel()
-
-        addSubview(logoutButton)
+        
+        if let _ = KeychainAccess().retrieveToken() {
```

Comment:
Is there a reason why we're calling `retrieveToken` but not using its returned value?

Reply:
we're seeing if the token exists so we can set up the logout button

Reply:
kk